spec/helpers/admin_helper_spec.rb

```diff
@@ -0,0 +1,59 @@
+require 'rails_helper'
+
+RSpec.describe AdminHelper, :type => :helper do
+  describe '.get_memento_uri_from_handler' do
+    it 'returns the memento uri based on the handler value' do
+      handler = "--- !ruby/struct:Was::ThumbnailService::Capture::CaptureJob\nmemento_id: 100\ndruid_id: ab123cd4567\n"
```

Comment:
This stubbing here feels non-standard. I wonder if this is easier tested if moved into a model?